test/statetest/statetest.hpp

```diff
@@ -59,8 +63,54 @@ struct StateTransitionTest
     std::unordered_map<uint64_t, std::string> input_labels;
 };
 
+template <typename T>
+T from_json(const json::json& j) = delete;
+
+template <>
+state::BlockInfo from_json<state::BlockInfo>(const json::json& j);
+
+template <>
+state::State from_json<state::State>(const json::json& j);
+
+template <>
+state::Transaction from_json<state::Transaction>(const json::json& j);
+
 StateTransitionTest load_state_test(const fs::path& test_file);
 
 void run_state_test(const StateTransitionTest& test, evmc::VM& vm);
 
+namespace utils
+{
+
+inline std::string shortest_hex(uint64_t v, bool add_prefix = true)
+{
+    if (v == 0)
+    {
+        return add_prefix ? "0x0" : "0";
+    }
+
+    std::string value;
+    for (auto i = 0; i < 8; ++i)
+        value += evmc::hex(static_cast<uint8_t>(v >> 8 * (8 - i - 1)));
```

Comment:
Hm, can't this actually be done with?
```suggestion
    value = intx::hex(intx::uint256{v});
```

Reply:
Done